Cargo.toml

```diff
@@ -17,6 +17,7 @@ exclude = [
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
+base64-stream = "1.2.7"
```

Comment:
TBH I'm a little surprised that the `base64` crate (which we already depended on transitively through `reqwest`) doesn't support this, but it does seem like it only has a [decoding reader](https://docs.rs/base64/0.21.0/base64/read/index.html) and an [encoding writer](https://docs.rs/base64/0.21.0/base64/write/index.html), with no decoding writer or encoding reader.

Doing an review of `base64-stream` itself might be worthwhile as it looks like it contains some unsafe code but is fairly digestible, but we don't require every new dependency be audited yet for cargo-vet.

Reply:
I was also surprised. I think the `base64` crate should expose structs which store the state needed for streamed encoding/decoding and allow explicit data IO so you don't necessarily need to have a `Read` or `Write`, too (which would make support for async stuff easier). Oh well.

I can review `base64-stream` (I've already read over a few files in it anyway).